Source/Blazorise/Utilities/Formatters/Parsers.cs

```diff
@@ -64,8 +64,10 @@ public static class Parsers
     public static readonly string[] SupportedParseDateTimeFormats = new string[]
     {
         InternalDateTimeFormat,
+        $"{InternalDateFormat}:ss",
```

Comment:
What is this one good for? this will result in `yyyy-MM-dd:ss`. Is there a case when this will be happening?